src/utils/editor/source-editor.js

```diff
@@ -16,6 +16,10 @@ require("codemirror/mode/elm/elm");
 require("codemirror/mode/clojure/clojure");
 require("../../components/Editor/codemirror-mozilla.css");
 require("codemirror/addon/search/searchcursor");
+require("codemirror/addon/fold/foldcode");
+require("codemirror/addon/fold/brace-fold");
+require("codemirror/addon/fold/indent-fold");
```

Comment:
I can add `xml-fold` as well that may be a nice touch for HTML files (and maybe JSX? haven't tested this though), can also add `comment-fold` too.